systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -434,4 +411,49 @@ public static void copyImagePullSecret(String namespace) {
                 .withData(Collections.singletonMap(".dockerconfigjson", pullSecret.getData().get(".dockerconfigjson")))
                 .build());
     }
+
+    private static JsonArray getJsonArrayFromLog(String log) {
```

Comment:
Would be great to put here some JavaDoc what is the excepted structure of the input `log` :)